packages/ui-contract-editor/README.md

```diff
@@ -116,7 +116,12 @@ clauseProps = {
 }
 ```
 
----
+The functions in `clauseProps` like `CLAUSE_DELETE_FUNCTION`,`CLAUSE_EDIT_FUNCTION` and `CLAUSE_TEST_FUNCTION`  are not implemented by default because they typically require interaction with other components or the persistence layer of the embedding application.
+These functions will be supplied by the user of the component.
+For more information in realtion to the `props` , `values` and `functionality` of Contract Editor visit [README.md](./src/ContractEditor/README.md)
```

Comment:
One last change, I think we should remove this last line since that `README` file no longer exists.

Reply:
Also keep the `---` horizontal rule just after this section you added.